backtesting_graphs/_stats.py

```diff
@@ -35,8 +32,8 @@ def geometric_mean(returns: pd.Series) -> float:
 def compute_stats(
         trades: Union[List['Trade'], pd.DataFrame],
         equity: np.ndarray,
+        calc_df: pd.DataFrame,
         ohlc_data: pd.DataFrame,
-        strategy_instance: 'Strategy',
         risk_free_rate: float = 0,
 ) -> pd.Series:
     assert -1 < risk_free_rate < 1
```

Comment:
Could add docstrings here...what are your thoughts? (I've been obsessed with Docstrings this week haha)